qemu/tests/cfg/multi_disk_random_hotplug.cfg

```diff
@@ -81,6 +81,8 @@
     variants:
         - @serial:
         - parallel:
+            Windows:
+                interval_time_unplug = 60
```

Comment:
This issue related to scsi, suggest adding scsi filter in case suppress some product bug.

Reply:
you mean just run `parallel` case on virtio-scsi? no need virtio-blk?

Reply:
sorry, I got you, will update.

Reply:
Hi @qingwangrh 
thanks, have updated.